### ISSUE TITLE
Should we still support reverse engineering?

### ISSUE THREAD

**Issue by dudadornelles** [gocd-contrib/gomatic#25]
Just want to hear the ideas out there;

There is a lot of effort that went into supporting reverse engineering of pipelines and to keep supporting it we need to be a lot more thorough reviewing PRs and spend time testing and making sure that all new things are reversible. At the same time, we have a lot to catch up: config repos, auth plugins, roles.

Is reverse engineering a feature we want to keep in gomatic? Is the effort on maintaining it compatible with the value it brings?

**Comment by alanjds** [comment]
Just to say, I starred this repo instead of similar ones because of the reverse engineering.

**Comment by gkirchner** [comment]
Yes please, stay with reverse engineering! It is a very valuable feature to our project.
